atomicfu-common/src/main/kotlin/kotlinx/atomicfu/Trace.kt

```diff
@@ -0,0 +1,45 @@
+package kotlinx.atomicfu
+
+import kotlin.js.JsName
+
+/**
+ * Creates Trace object for tracing atomic operations.
+ *
+ * Usage: create a separate field for trace and pass to atomic factory function:
+ *
+ * ```
+ * val trace = trace(size)
+ * val a = atomic(initialValue, trace)
+ * ```
+ */
+@JsName("atomicfu\$trace\$")
+fun trace(size: Int = 32, format: (AtomicInt, String) -> String = { index, text -> "$index: $text" }): Trace = TraceImpl(size, format)
+
+val NO_TRACE = Trace()
+
+/**
+ * Default no-op Trace implementation that can be overridden
+ */
+public open class Trace {
+
+    @JsName("atomicfu\$trace\$append\$")
```

Comment:
Seems, that I should create ```expected interface Trace {..}``` , then actual declarations of `Trace` in modules for every platform and move these @JsName annotations to JS module (to support Trace on Native too).
Is there a way to add JS annotations to common code?